pyrene/src/components/TabView/TabView.jsx

```diff
@@ -112,6 +113,7 @@ export default class TabView extends React.Component {
                 role="tab"
               >
                 {tab.name}
+                {tab.nameRenderCallback && tab.nameRenderCallback()}
```

Comment:
you can also do [that way](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) :

`{tab.nameRenderCallback?.()}`

Reply:
D'awwww! Indeed! Thanks a lot :) Yes it's more elegant that way :)